docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM adoptopenjdk:11-jre-hotspot
```

Comment:
adding the comment here, but really it pertains to the folder structure. i would probably call the folder "cli" or "client" rather than docker. within the folder we also have a dockerfile and can build a docker image, and that's ok, but the tool per se could be built standalone without docker, right?